emulator/src/core/emulator/instructions/memory.rs

```diff
@@ -0,0 +1,203 @@
+use super::super::{Emulator, EResult};
+
+impl Emulator {
+    pub fn stax(&mut self, register: &str) -> EResult<()> {
```

Comment:
Is there a reason for every method in this class to return `EResult<()>` and never actually return anything?

Reply:
Agree, 99% of these are pretty useless... just copied it around 🤷🏼‍♂️ @moorts is there a specific reason why we should use it?

Reply:
No, not really. They can be removed, originally I just used EResult<()> for things that access RAM so i can do bounds checking, but I think I didn't actually do that anywhere... Anyways when RAM isn't accessed inside the method, or no errors of different nature can occur, remove these by all means.